pages/account.tsx

```diff
@@ -359,42 +320,42 @@ const Account: NextPage = () => {
             p={{
               base: '2.5rem 1rem',
               md: 10
-            }}            
+            }}
             w={{
               base: 'calc(100vw - 64px)',
               lg: '1044px'
             }}
           >
             <VStack spacing={10}>
               <Box>
-              <Text
-                color='white'
-                fontSize='2xl'
-                textAlign={'center'}
-              >Verified Addresses</Text>
-              <Text
-                color='white'
-                w='25rem'
-                mx={{
-                  base: 'auto',
-                  md: 'unset'
-                }}
-                textAlign={{
-                  base: 'center',
-                  md: 'unset'
-                }}
-              >The token balances in these addresses will all be used during the voting process.</Text>
+                <Text
+                  color='white'
+                  fontSize='2xl'
+                  textAlign={'center'}
+                >Verified Addresses</Text>
+                <Text
+                  color='white'
+                  w='25rem'
+                  mx={{
+                    base: 'auto',
+                    md: 'unset'
+                  }}
+                  textAlign={{
+                    base: 'center',
+                    md: 'unset'
+                  }}
+                >The token balances in these addresses will all be used during the voting process.</Text>
               </Box>
               {/* Render Poll Listings */}
               <Box
                 overflowX={'scroll'}
                 w='100%'
               >
-              <DataTable
-                data={addressesData.filter((addressData: any) => addressData.verifiedDate !== 'False')}
-                columns={columns}
-                loading={isLoadingAddresses}
-              />
+                <DataTable
+                  data={addressesData.filter((address: any) => address.verifiedDate !== 'False')}
+                  columns={columns}
+                  loading={isLoadingAddresses}
+                />
```

Comment:
locally, I had removed this. This will make it so unverifiect address don't appear in the table.